desktop/ui/src/main/java/org/datacleaner/windows/AnalysisJobBuilderWindowImpl.java

```diff
@@ -505,10 +512,10 @@ public void updateStatusLabel() {
                     setStatusLabelWarning();
                 }
             } catch (Exception ex) {
+                executeable = false;
```

Comment:
This setting to false was in "if" and "else if" branches but not in the "else" branch. I looked into GIT history and it has never been there. But @LosD said that job execution was probably blocked in the past. It means it was achieved in a different way which I don't know. 

And secondly, is there any reason why this last "else" branch did not set "executable" to false? Or is that just something that was forgotten?